pages/pixel-ui/avatar.mdx

```diff
@@ -0,0 +1,22 @@
+import MethodList from "/components/method_list";
+
+# Avatar
+## Methods

```

Comment:
A new section should be added above the methods which details the Panel which this element derives from. See the following page under the "Parent" section as an example: https://wiki.facepunch.com/gmod/DLabel#Parent